lib/src/utilities/numbers.dart

```diff
@@ -15,6 +15,7 @@ double? parseDouble(String? rawDouble, {bool tryParse = false}) {
       .replaceFirst('em', '')
       .replaceFirst('ex', '')
       .replaceFirst('px', '')
+      .replaceFirst('pt', '')
```

Comment:
So I think I've been overthinking this.

I think we can just use CSS's rule here and just multipy `pt` values by 1.33333 to get pixels. This isn't quite the right fix though.

Reply:
Can you take a look at https://github.com/dnfield/vector_graphics/pull/114/files?

We'd want a similar fix here.

Reply:
Are you interested in doing this? If not I'll push up a fix.